src/libLookup/Lookup.cpp

```diff
@@ -133,18 +133,24 @@ void Lookup::InitSync() {
           m_mediator.m_blocklinkchain.GetLatestIndex() + 1, true,
           LOOKUP_NODE_MODE);
       GetTxBlockFromSeedNodes(txBlockNum, 0);
-
-      this_thread::sleep_for(chrono::seconds(NEW_NODE_SYNC_INTERVAL));
-    }
-    // Ask for the sharding structure from lookup
-    ComposeAndSendGetShardingStructureFromSeed();
-    std::unique_lock<std::mutex> cv_lk(m_mutexShardStruct);
-    if (cv_shardStruct.wait_for(
-            cv_lk, std::chrono::seconds(GETSHARD_TIMEOUT_IN_SECONDS)) ==
-        std::cv_status::timeout) {
-      LOG_GENERAL(WARNING, "Didn't receive sharding structure!");
-    } else {
-      ProcessEntireShardingStructure();
+      std::unique_lock<std::mutex> cv_lk(m_mutexCvSetRejoinRecovery);
+      if (m_mediator.m_lookup->cv_setRejoinRecovery.wait_for(
```

Comment:
This cv will wait for long time until timedout in scenarios like:
- No txblks msg received from upper_seed
- `GetDSInfo` in `CommitTxBlocks` failed. Basically node didn't marked syncType as `NO_SYNC`.